doc/kd-release-process.md

```diff
@@ -98,7 +103,7 @@ Delete the x.y.z-release branch everywhere (local, your GitHub, main GitHub).
 
 #### ADVERTISE THE RELEASE
 
-Modify the "current" wiki page documenting each CR as necessary to indicate that it is documenting a released API version. If development begins on a new API version in the future this text should be changed at that time.
+Modify the wiki page documenting each CRD so that it includes documentation of the support for any new properties added in this release. (Reference the existing tables to see how properties for a new version are added and marked.) Note that this is assuming the K8s API versioning practice where new properties are added to an existing API version while maintaining backwards compatibility; if we reach the point where a new API version is required then the CRD documentation will have to be further restructured.
```

Comment:
Can any contributor create release?

Reply:
Nope, this is for those of us that have repo write and wiki write access. (And at the moment, really just for me.)